src/internal/m365/service/sharepoint/api/pages.go

```diff
@@ -68,9 +67,13 @@ func GetSitePages(
 				err  error
 			)
 
-			page, err = serv.Client().SitesById(siteID).PagesById(pageID).Get(ctx, opts)
+			page, err = serv.
+				Client().
+				SitesById(siteID).
+				PagesById(pageID).
+				Get(ctx, opts)
 			if err != nil {
-				el.AddRecoverable(ctx, graph.Wrap(ctx, err, "fetching page"))
+				el.AddRecoverable(ctx, clues.Wrap(err, "fetching page"))
```

Comment:
Could you please remind me what the expectation is here for clues injection? Are we expecting the middleware layers to add that? If not then pretty much all of the calls like this in the files in this PR will need to add clues themselves

Reply:
It's handled with the graph service client itself.  So long as the creation of the client uses our standard adapter creation func (it may not here, but pages aren't supported either), we're good.